.travis.yml

```diff
@@ -6,10 +6,10 @@ services:
 env:
 - TORTUGA_BUILD_DOCKER="true"
 before_install:
-- docker pull univa/tortuga-build-kit:master-latest
+- docker pull univa/tortuga-build-kit:7.0.2-devel-latest
```

Comment:
I think that this should probably be changed to `7.0.2` as `-devel` is going away soon.